Dockerfile

```
@@ -0,0 +1,8 @@
+FROM python:3.8
+RUN mkdir -p /etc/migrations
+WORKDIR /etc/migrations
+COPY alembic.ini alembic.ini
+COPY alembic alembic
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+CMD ['alembic', 'upgrade', 'head']
```

```suggestion
CMD ["alembic", "upgrade", "head"]
```
Wasn't able to run 
```
docker build . -t alembic
docker run -v $(pwd)/.env:/etc/migrations/.env --link db:db alembic
```
on MacOS with '.' instead of "."

---

Thanks! Hope other things work now